src/Valit/ValitRuleBooleanExtensions.cs

```diff
@@ -0,0 +1,32 @@
+namespace Valit
+{
+  public static class ValitRuleBooleanExtensions
+  {
+    public static IValitRule<TObject, bool> IsTrue<TObject>(this IValitRule<TObject, bool> rule) where TObject : class
+        => rule.Satisfies(p => p);
+
+    public static IValitRule<TObject, bool?> IsTrue<TObject>(this IValitRule<TObject, bool?> rule) where TObject : class
```

Comment:
I think that this Satisfies also requires p.HasValue

Reply:
👍 changed it.